fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -765,7 +778,7 @@ protected void process() {
      * @param groupName
      */
     protected void grantPrivileges(String groupName) {
-        if (dbType == DbType.DERBY) {
+        if (dbType == DbType.DERBY || dbType == DbType.POSTGRESQL) {
             return;
```

Comment:
no grants?
If we are not implementing here, we should create a backlog issue.

Reply:
yes, not needed in my test for the postgresql support.
according to my instruction, fhirserver user is grant full access to the database, and also create/update schema uses fhirserver user. so seems no access issue for postgresql so far.

Reply:
added issue #946 for this